parser/parser_manager.go

```diff
@@ -32,6 +32,19 @@ type UpdateFile struct {
 	Signature []byte
 }
 
+type UpdateData struct {
+	// path to directory containg update data
+	Path string
+	// update data files
```

Comment:
Or not. This is actually a path to the directory that might contain not only data files. There might be `meta-data`, `type-info` files and so on. I think it will be a bit misleading to say that this is where data files are.